apps/ewallet/lib/ewallet/web/paginators/start_after_paginator.ex

```diff
@@ -0,0 +1,375 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule EWallet.Web.StartAfterPaginator do
+  @moduledoc """
+  The StartAfterPaginator allows querying of records by specified `start_after` and `start_by`.
+  They take in a query, break the query down,
+  then selectively query only records that are after the given `start_after`'s scope.
+
+  If the `start_after` is nil, then return records from the beginning.
+
+  For example:
+
+  ```
+  StartAfterPaginator.paginate_attrs(
+    Account,
+    %{"start_after" => "acc_3", "start_by" => "id", "per_page" => 10},
+    [:id]
+  )
+  ```
+
+  Let's say we have 10 accounts with ids:
+  ["acc_1", "acc_2", "acc_3", ... , "acc_10"]
+
+  The code above return a pagination with accounts:
+  ["acc_4", "acc_5", "acc_6", ... ,"acc_10"]
+
+  Note that an account with id "acc_3" is not included because the query range is exclusive.
+
+  However, query accounts from the beginning is possible by specifying `start_after` to nil:
+
+  ```
+  StartAfterPaginator.paginate_attrs(
+    Account,
+    %{"start_after" => nil, "start_by" => "id", "per_page" => 10},
+    [:id]
+  )
+  ```
+
+  Return accounts:
+  ["acc_1", "acc_2", "acc_3", ... ,"acc_10"]
+  """
+  import Ecto.Query
+  alias EWalletDB.Repo
+  alias EWallet.Web.Paginator
+
+  @doc """
+  Paginate a query by attempting to extract `start_after`, `start_by` and `per_page`
+  from the given map of attributes and returns a paginator.
+  """
+  @spec paginate_attrs(Ecto.Query.t() | Ecto.Queryable.t(), map(), Ecto.Repo.t()) ::
+          %Paginator{} | {:error, :invalid_parameter, String.t()}
+  def paginate_attrs(queryable, attrs, allowed_fields \\ [], repo \\ Repo)
+
+  # Prevent non-string, non-atom `start_by`
+  def paginate_attrs(_, %{"start_by" => start_by}, _, _)
+      when not is_binary(start_by) and not is_atom(start_by) do
+    {:error, :invalid_parameter, "`start_by` must be a string"}
+  end
+
+  def paginate_attrs(queryable, %{"sort_by" => "created_at"} = attrs, allowed_fields, repo) do
```

Comment:
Maybe you can re-use the `default_mapped_fields` from `apps/ewallet/lib/ewallet/web/orchestrator.ex` instead of re-defining it here